File: pseudo.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

void concat(char string[], char sn[], int cin);

int main()
{
	char pseudo[]="";
	char short_name[]="ABC";
	char string[]="";
	int cin=1234;
	concat(string,short_name,cin);
	strcpy(pseudo,string);
	printf("%s\n",pseudo);
	return 0;
}

void concat(char string[],char sn[], int cin)
{
	char scin[5];
	sprintf(scin,"%d",cin);
	strcpy(string,sn);
	strcat(string,scin);
}

File: pseudo.py
import short_name

#short_name="ABC"
short_name=short_name.get_short_name("mahmoud ben salah")
cin="1234"

def concat(sn,cin):
	return (sn+str(cin))

if __name__=="__main__":
	print(concat(short_name,cin))
